docs/src/changelog.md

```diff
@@ -19,6 +19,10 @@ If your currently installed *Quality-time* version is not the latest version, pl
 - When measuring test cases with Visual Studio TRX as source, search all test category items for test case ids, instead of cutting the search short after the first match. Fixes [#10460](https://github.com/ICTU/quality-time/issues/10460).
 - Correctly parse empty Axe-core JSON report. Fixes [#10487](https://github.com/ICTU/quality-time/issues/10487).
 
+### Changed
+
+- Support for Checkmarx CxSAST as source for metrics is deprecated. Closes [#10383](https://github.com/ICTU/quality-time/issues/10383).
```

Comment:
Perhaps "deprecated and marked for removal in the future" is more direct, but I'm not sure you want it to be so